.drone.star

```diff
@@ -1123,50 +1128,51 @@ def e2eTests(ctx):
         "temp": {},
     }]
 
-    e2e_test_ocis = [{
-        "name": "e2e-tests",
-        "image": OC_CI_NODEJS % DEFAULT_NODEJS_VERSION,
-        "environment": {
-            "BASE_URL_OCIS": "ocis-server:9200",
-            "HEADLESS": "true",
-            "OCIS": "true",
-            "RETRY": "1",
-            "WEB_UI_CONFIG": "%s/%s" % (dirs["base"], dirs["ocisConfig"]),
-            "LOCAL_UPLOAD_DIR": "/uploads",
-            "API_TOKEN": "true",
-        },
-        "commands": [
-            "cd %s" % dirs["web"],
-            "sleep 10 && pnpm test:e2e:cucumber tests/e2e/cucumber/**/*[!.oc10].feature",
-        ],
-    }]
+    pipelines = []
 
-    e2eTestsSteps = \
-        skipIfUnchanged(ctx, "e2e-tests") + \
-        restoreBuildArtifactCache(ctx, "ocis-binary-amd64", "ocis/bin/ocis") + \
-        restoreWebCache() + \
-        restoreWebPnpmCache() + \
-        tikaService() + \
-        ocisServer("ocis", 4, [], tika_enabled = True) + \
-        e2e_test_ocis + \
-        uploadTracingResult(ctx) + \
-        logTracingResults()
-
-    if ("skip-e2e" in ctx.build.title.lower()):
-        return []
```

Comment:
These lines could be left here, before any of the real logic.
If we want to `skip-e2e` then we can return straight away. There is no need to get into the `for` loop.

Reply:
Ahh, I mistakenly copied that also inside the for loop